src/components/Comments/hooks/useCommentAction.js

```diff
@@ -40,14 +44,13 @@ const useCommentAction = () => {
         }
       }
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'eman');
-      }
+      console.log(e, 'eman');
     }
   };
 
   return {
-    showAlertDelete
+    showAlertDelete,
+    onDeleteCommentClicked
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's an issue with error handling. In the catch block, you're logging the error to the console without checking if the environment is development. This could potentially expose sensitive information in a production environment. Also, `onDeleteCommentClicked` function is now returned from the hook but it was not being returned before. If this function is not used outside of this hook, there's no need to return it.

```diff
     } catch (e) {
-      console.log(e, 'eman');
+      if (__DEV__) {
+        console.log(e, 'eman');
+      }
     }
   };
 
   return {
-    showAlertDelete,
-    onDeleteCommentClicked
+    showAlertDelete
   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->